src/Core/Prefab/PlanarView.js

```diff
@@ -55,12 +55,12 @@ class PlanarView extends View {
         placement.heading = placement.heading || 0;
         placement.range = placement.range || max;
 
+        CameraUtils.transformCameraToLookAtTarget(this, camera3D, placement);
```

Comment:
it's out of PR scope?

Reply:
The placement of this line caused an issue with the `Y` key control in 2.5D planar example (the key is supposed to move the camera to its initial position, and without this change, the initial position was coordinates `(0, 0, 0)`).
Since it is related to key controls, I figured perhaps mistakenly this change could relate to the PR subject.